docs/source/markdown/podman-pod-create.1.md

```diff
@@ -141,7 +141,8 @@ Set network mode for the pod. Supported values are:
 
 #### **--network-alias**=strings
 
-Add a DNS alias for the container. When the container is joined to a CNI network with support for the dnsname plugin, the container will be accessible through this name from other containers in the network.
+Add a DNS alias for the container. When the container is joined to a CNI network with support for the dnsname plugin, the container will be accessible through this name from other containers in the network.  NOTE: A container will only have access to aliases on the first network that it joins.  This is a limitation
```

Comment:
This is the podman pod create man page, so the sentence is not entirely correct. This sentence must be added to the podman create/run man page.

For podman pod create you should rewrite so something like this:

Add a DNS alias for the pod. When the pod is joined to a CNI network with support for the dnsname plugin, the containers inside the pod will be accessible through this name from other containers in the network.

Reply:
Good catch, reverted and updated elsewhere, TYVM!